tools/bin/ariaAOIassist.py

```diff
@@ -32,8 +32,8 @@ def createParser():
             help='Specify directory to deposit all outputs. Default is local directory where script is launched.')
     parser.add_argument('-t', '--tracks', dest='tracks', type=str, default='all',
             help='Include only specified track number in results. Can be multiple, separated by spaces. Default : All')
-    parser.add_argument('-l', '--lat_bounds', dest='latBounds', type=str, default='-75 75',
-            help='Specify a search for only frames that fall within these lat bounds. Default : -75 75')
+    parser.add_argument('-l', '--lat_bounds', dest='latBounds', type=str, default=None,
+            help='Specify a search for only frames that fall within these lat bounds. Default : None')
```

Comment:
@rzinke can you rebase your PR. Im not sure, but this seems to undo what you had before.

Reply:
Hi David, I meant to remove the default values for lat_bounds. Now there should be no default bounds. 
Could you give this a try with your data to confirm?